resources/jenkins/setup_jenkins.bat

```diff
@@ -0,0 +1,5 @@
+mkdir "%ProgramFiles%\jenkins"
+copy C:\vagrant\resources\jenkins\jenkins.war "%ProgramFiles%\jenkins"
+copy C:\vagrant\resources\jenkins\start_jenkins.bat "%ProgramFiles%\jenkins"
```

Comment:
The start_jenkins.bat file isn't found in the resources\jenkins folder. Did you forget to check the file in?